tests/test_simple_put.cpp

```diff
@@ -124,6 +124,11 @@ void test_simple_put_core(const char * fileName,
   snowflake_term(sf); // purge snowflake context
 }
 
+static int donothing(void **unused)
```

Comment:
I don't see where this function is used

Reply:
cmocka_unit_test_teardown(test_simple_put_skip, donothing),   
cmocka_unit_test_teardown(test_simple_get, teardown)

teardown delete all the data in the table. To download the file in the subsequent call we should not teardown in the put call and teardown in the get call.